src/UI/ProgressSteps/index.tsx

```diff
@@ -30,12 +33,17 @@ export const ProgressSteps = ({
 
   return (
     <div
-      className={`progress-steps d-flex justify-content-center ${className}`}
+      className={`${globalStyles.progressSteps} ${globalStyles.flexRow} ${globalStyles.justifyContentCenter} ${className}`}
```

Comment:
Could you please try and enumerate the styles inside an array or directly as arguments? I think it will look much much better